docs/source/build.md

```diff
@@ -369,8 +364,7 @@ find_and_configure_raft(VERSION    ${RAFT_VERSION}.00
         COMPILE_LIBRARIES        NO
         USE_NN_LIBRARY           NO
         USE_DISTANCE_LIBRARY     NO
-        ENABLE_NN_DEPENDENCIES   NO  # This builds FAISS if not installed
-        USE_FAISS_STATIC         NO
+        ENABLE_NN_DEPENDENCIES   NO
```

Comment:
We should remove the `ENABLE_NN_DEPENDENCIES` option altogether now, since this was only used to specify whether FAISS should by included in the dependencies.